packages/lib/src/asset-manager/asset-manager.wc.svelte

```diff
@@ -73,9 +79,14 @@
     return Promise.all(
       Array.from(files)
         .filter((file) => {
+          const fileExists = items.some((item) => item.name === file.name && item.size === file.size);
+          if (fileExists) {
+            console.log(`File ${file.name} already exists.`);
```

Comment:
Try to find a place where to display errors instead of just console logging since this isn't visible to the user.